docs/source/backends/airos.rst

```diff
@@ -0,0 +1,418 @@
+=============
+AirOS Backend
+=============
+
+.. include:: ../_github.rst
+
+The ``AirOs`` backend allows to generate AirOS v8.3 compatible configurations.
+
+.. warning::
+
+    This backend is in experimental stage: it may have bugs and it will
+    receive backward incompatible updates in during the first 6 months
```

Comment:
argh, sorry, I suggested you a wrong description, it should be `it will receive backward incompatible updates during the first 6 months` (remove `in`)